R/utils-html.R

```diff
@@ -92,8 +92,8 @@ printReportFeature <- function(feature, isTable=FALSE, m=NULL, mar_values=c(8, 3
 formatComments <- function(comments){
   split_comments <- unlist(comments)
   if(isEmptyOrBlank(split_comments)){return(split_comments)}
-  htmlbreaks_inside <- lapply(split_comments, gsub, pattern="\r\n", replacement="</br>")
-  htmlbreaks_end <- lapply(htmlbreaks_inside, paste0, "</br>", collapse="")
+  htmlbreaks_inside <- lapply(split_comments, gsub, pattern="\r\n", replacement="<br/>")
```

Comment:
according to this http://www.w3schools.com/code/tryit.asp?filename=FBNJO5J8MUTJ

this works, but it could only be working because the browser is allowing it to.